server/views/referrals/new/additionalInformation/show.njk

```diff
@@ -65,12 +65,22 @@
           errorMessage: errors.messages.additionalInformation
         }) }}
 
-        {{ govukButton({
+        <div class="govuk-button-group">
+          {{ govukButton({
           attributes: {
             'data-testid': 'additional-information-submit-button'
           },
           text: "Continue"
         }) }}
+          {{ govukButton({
+          attributes: {
+            'data-testid': 'additional-information-submit-button'
```

Comment:
`'additional-information-skip-button'` or similar. It doesn't look like it's being used currently in any associated cypress tests.